glove2word2vec.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2016 Manas Ranjan Kar <manasrkar91@gmail.com>
+# Licensed under the MIT License https://opensource.org/licenses/MIT
+
+"""
+CLI USAGE: python glove2word2vec.py <GloVe vector file> <Output model file>
+
+Convert GloVe vectors into Gensim compatible format to instantiate from an existing file on disk in the word2vec C format;
```

Comment:
The format is not "gensim format"; it is the format set by the original word2vec tool (which was written in C).